### DISCUSSION TITLE
Welcome to hltv-api Discussions!

### DISCUSSION THREAD

**Discussion by SocksPls** [SocksPls/hltv-api#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vladiwnl** [comment]
Hello! Need some advice, so.. 
my version of data collection for future modeling:
scrap game results by date, then getting _match, team and players_ info. 
But think its impossible to do as in get_results by date function there isnt any ID's. 

Any tips/suggestions to solve it? 
My thought that get_results_by_date function should have team id's to solve that.

**Reply by zGrav** [reply]
This can be easily added since we know who's who from the URL itself.

@SocksPls worth adding `ids` all around?

**Reply by zGrav** [reply]
> This can be easily added since we know who's who from the URL itself.
> 
> @SocksPls worth adding `ids` all around?

Adding this as we speak

**Reply by zGrav** [reply]
Done, opening MR.

**Reply by zGrav** [reply]
Merged and rolled

**Comment by grantephross** [comment]
Thank you for making this, the API that is published by PIP is terrible, this has so much more functionality :D

**Reply by zGrav** [reply]
@grantephross some data types changed with new update that I pushed, so if your bot starts complaining that might be why :stuck_out_tongue:

**Reply by grantephross** [reply]
my bot is a pretty simple discord and twitch bot that grabs upcoming match information for my friends team :) very simple ty for fixing the time/making it changeable via variable string.

**Reply by grantephross** [reply]
I'm really bad at writing code that isn't procedural so that was really helpful :P

**Comment by JoeMCodes** [comment]
Hi, does anyone know what the lxml==4.9.1 importa ctually does in the package? It seems to work fine without it.

**Reply by SocksPls** [reply]
lxml is a faster HTML parser than the `html.parser` included in bs4
